.github/workflows/dart.yml

```diff
@@ -16,6 +37,7 @@ jobs:
         # Test latest stable, beta, dev, and main channels + last stable release
```

Comment:
Please update the comment here to describe the `raw` flag below.

Reply:
done, thanks!